frontend/language/src/nb.json

```diff
@@ -1535,10 +1534,10 @@
   "ux_editor.component_title.Header": "Tittel",
   "ux_editor.component_title.IFrame": "IFrame",
   "ux_editor.component_title.Image": "Bilde",
-  "ux_editor.component_title.Input": "Kort svar",
-  "ux_editor.component_title.InstanceInformation": "Informasjon om instans",
-  "ux_editor.component_title.InstantiationButton": "Instansieringsknapp",
-  "ux_editor.component_title.Likert": "Likert",
+  "ux_editor.component_title.Input": "Lite tekstfelt",
```

Comment:
Jeg vet ikke hva som er best, men ordet "lite" gir meg inntrykk av at det er et lite tekstfelt. Det er vel ikke nødvendigvis lite? "Kort svar" synes jeg gir meg informasjon om bruk, altså denne komponenten brukes for å hente inn korte svar. Ikke lange avhandlinger. Det er min tolkning, men vet ikke om det er rett eller om flere tenker sånn. 😅

Reply:
> Jeg vet ikke hva som er best, men ordet "lite" gir meg inntrykk av at det er et lite tekstfelt visuelt. Det er vel ikke nødvendigvis lite visuelt? "Kort svar" synes jeg gir meg mer informasjon om bruken, altså denne komponenten brukes for å hente inn korte svar, og ikke lange avhandlinger. Det er min tolkning, men vet ikke om det er rett eller om flere tenker sånn. 😅

Det kan være en relevant diskusjon. I norsk IT-terminologi heter input field - inndatafelt, text field - tekstfelt og text area - tekstområde. Det kan kanskje være et alternativ å holde seg til tekstfelt og tekstområde? Legg gjerne inn en kommetar i regnearket om begreper, så kan vi ta det med i en av beslutningsøktene våre i høst.